src/plonk/permutation.rs

```diff
@@ -35,8 +35,8 @@ impl Argument {
         // l_0(X) * (1 - z(X)) = 0
         //
         // degree columns + 1
-        // z(X) \prod (p(X) + \beta s_i(X) + \gamma)
-        // - z(omega^{-1} X) \prod (p(X) + \delta^i \beta X + \gamma)
+        // z(omega X) \prod (p(X) + \beta s_i(X) + \gamma)
+        // - z(X) \prod (p(X) + \delta^i \beta X + \gamma)
```

Comment:
```suggestion
        // - z(X) \prod (p(X) + \beta \delta^i X + \gamma)
```
This is how it's written in the halo2 book.

Reply:
Thanks. Not sure that I like this TBH, because it's important not to lose sight of powers of `\delta` here which are really doing something; the \beta term is a challenge that basically behaves like a constant at this point in the protocol.

Reply:
OK, we can change the book instead.